README.md

```diff
@@ -40,24 +40,21 @@ Run this command to install the SDK in your project:
 ```bash
 npm install --save @asyncapi/generator-model-sdk
 ```
-
 ## How it works
 
-The process of creating data models from input data consists of three main process: transformation input data to JSON Schema, simplification and generation.
-
-### The transformation process
-
-The transformation process starts from checking what type of input data is provided and based on that it is processed differently. Currently JSON Schema Draft 7 and AsyncAPI version 2.0.0 are supported. It is also in this stage that references are resolved and schemas are named. Each input data must be converted to JSON Schema or be a subset of JSON Schema (like AsyncAPI spec) to go to the next process - the simplification stage.
+The process of creating data models from input data consists of 2 processes, the input and generation process.
 
-### The simplification process
+### The input process
 
-In order to simplify the model generation process as much as possible, AsyncAPI Model SDK simplifies the transformed JSON schema into `CommonModel`s, which contain information about the type of model it is, what properties it might have, etc, so finally we have a bare minimum schema. Think of it like converting data validation rules (JSON Schema) to a data definition.
+The input process ensures that any supported input are handles correctly, the basics is that any input needs to be converted into our internal model representation `CommonModel`. The following inputs which are supported:
```

Comment:
Few lang typos here:
```suggestion
The input process ensures that any supported input is handled correctly, the basics are that any input needs to be converted into our internal model representation `CommonModel`. The following inputs are supported:
```

I would like to suggest using https://app.grammarly.com so most of those will be fixed. I'm not a native English speaker either so I usually do the same mistakes.

Reply:
Thanks for the review 🙇 It is now installed just to catch the most obvious errors 😄